src/App.tsx

```diff
@@ -23,24 +23,6 @@ function getLibrary(provider: any): Web3Provider {
 }
 function App() {
   const context = useWeb3React<Web3Provider>();
```

Comment:
Do we need this line or can it be removed or commented out for now?
```src/App.tsx
  Line 25:9:  'context' is assigned a value but never used  @typescript-eslint/no-unused-vars```

Reply:
On it now